src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -2012,16 +2024,17 @@ public SocketError SendFileAsync(SafeFileHandle fileHandle, long offset, long co
                 return errorCode;
             }
 
-            var operation = new SendFileOperation(this)
+            var operation = new SendFileOperation<TState>(this)
```

Comment:
On Unix we don't cache this operation.
[Other operations](https://github.com/dotnet/runtime/blob/664b96289bbcbb6abda6d2137e701c733f1a1094/src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs#L41-L45) are cached, the Windows-counterpart is cached too.

Shall it be cached here too or is (re-) creating just fine?